src/client/components/editor/index.js

```diff
@@ -338,6 +338,14 @@ class Editor extends React.Component {
 
   componentDidMount(){
     this.data.mountDeferred.resolve();
+    let doc = this.data.document;
+
+    let docs = JSON.parse(localStorage.getItem('my-factoids')) || [];
+    let docData = { id: doc.id(), secret: doc.secret(), name: doc.name() };
+    if( _.find(docs,  docData) == null ){
```

Comment:
What about the case where a user first views the document publicly and then privately?  It might make sense to merge each time.

Reply:
If they have visited both, why not give two links so that they know one is read only and one is write only?  The current non-debug seed flow doesn't mention any concept related to read/write links.

Reply:
In that case, it would make sense to have different UI elements for each listed document to allow opening in public and (when available) private mode.  Having a doc listed twice could be confusing later on.

Anyway, real users probably won't be using this page for now.  It's useful for demos  though.

The merging option is a bit simpler, since you don't need any UI changes.